l5kit/l5kit/tests/geometry/transform_test.py

```diff
@@ -2,7 +2,22 @@
 import pytest
 import transforms3d
 
-from l5kit.geometry import transform_point, transform_points
+from l5kit.geometry import transform_point, transform_points, transform_points_batch
```

Comment:
I wonder why not call `transform_points_batch` from `transform_points` to reuse code

Reply:
It's an option. The PR is still a draft in the end :)

Reply:
Now everything calls `transform_points_batch` :) I still need to fix tests to make them really useful (now testing one function against the other makes no sense anymore)